src/models/hit-types/item-hit.ts

```diff
@@ -78,7 +80,8 @@ export class ItemHit {
    * In bytes; computed during document construction, for collection items only.
    * Optional.
    */
-  @Memoize() get collection_size(): typeof Metadata.prototype.collection_size {
+  @Memoize()
+  get collection_size(): typeof SearchMetadata.prototype.collection_size {
     return this.rawMetadata?.fields?.collection_size != null
       ? new ByteField(this.rawMetadata.fields.collection_size)
```

Comment:
Looking through these, it occurs to me that some of these fields are properties of _search documents_ and not necessarily of actual _item metadata_. The distinction is probably not super meaningful given the way these are used, but figured it might be worth mentioning in case there's a desire to more clearly document the difference where these properties exist in the new item-metadata repo. (`num_favorites` is perhaps another example)